re-write some of the ewk corrections and fix few other typos

capalmer85 opened the pull request.

capalmer85 commented:
@mastrolorenzo I changed a few things about your implementation of the electroweak corrections. The biggest thing that you should note is that you should never add outfile->Fill() or add a SetBranch().  If you want to create a new branch, you just need to add a line in the cfg/newbranches.txt file.  Please look over this and let me know if you have some questions about this implementation.

Btw, I've tested.  It works, and I'm going to merge.

mastrolorenzo commented:
hi @capalmer,Many thanks for the clarification about the branch management! Concerning the modification you made, I have just went through them in the code and it is perfectly fine for me. 
cheersLuca



Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: capalmer85 <notifications@github.com> Date: 17/03/2018  02:37  (GMT+01:00) To: capalmer85/AnalysisTools <AnalysisTools@noreply.github.com> Cc: mastrolorenzo <luca.mastrolorenzo@cern.ch>, Mention <mention@noreply.github.com> Subject: Re: [capalmer85/AnalysisTools] re-write some of the ewk corrections and fix few other typos (#203) 
@mastrolorenzo I changed a few things about your implementation of the electroweak corrections. The biggest thing that you should note is that you should never add outfile->Fill() or add a SetBranch().  If you want to create a new branch, you just need to add a line in the cfg/newbranches.txt file.  Please look over this and let me know if you have some questions about this implementation.
Btw, I've tested.  It works, and I'm going to merge.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/capalmer85/AnalysisTools","title":"capalmer85/AnalysisTools","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/capalmer85/AnalysisTools"}},"updates":{"snippets":[{"icon":"PERSON","message":"@capalmer85 in #203: @mastrolorenzo I changed a few things about your implementation of the electroweak corrections. The biggest thing that you should note is that you should never add outfile-\u003eFill() or add a SetBranch().  If you want to create a new branch, you just need to add a line in the cfg/newbranches.txt file.  Please look over this and let me know if you have some questions about this implementation.\r\n\r\nBtw, I've tested.  It works, and I'm going to merge."}],"action":{"name":"View Pull Request","url":"https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373884628"}}}

capalmer85 commented:
Hi Luca. Very good.  Btw, I think that your code shouldn't have worked at
all because you re-declared the pointers locally in InitAnalysis().  Did
you check to see the output of the function before you made the PR?

On Sat, Mar 17, 2018 at 5:30 AM, mastrolorenzo <notifications@github.com>
wrote:

> hi @capalmer,Many thanks for the clarification about the branch
> management! Concerning the modification you made, I have just went through
> them in the code and it is perfectly fine for me.
> cheersLuca
>
>
>
> Sent from my Samsung Galaxy smartphone.
> -------- Original message --------From: capalmer85 <
> notifications@github.com> Date: 17/03/2018 02:37 (GMT+01:00) To:
> capalmer85/AnalysisTools <AnalysisTools@noreply.github.com> Cc:
> mastrolorenzo <luca.mastrolorenzo@cern.ch>, Mention <
> mention@noreply.github.com> Subject: Re: [capalmer85/AnalysisTools]
> re-write some of the ewk corrections and fix few other typos (#203)
> @mastrolorenzo I changed a few things about your implementation of the
> electroweak corrections. The biggest thing that you should note is that you
> should never add outfile->Fill() or add a SetBranch(). If you want to
> create a new branch, you just need to add a line in the cfg/newbranches.txt
> file. Please look over this and let me know if you have some questions
> about this implementation.
> Btw, I've tested. It works, and I'm going to merge.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
>
>
>
>
>
>
>
> {"api_version":"1.0","publisher":{"api_key":"
> 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> {"external_key":"github/capalmer85/AnalysisTools","title":"capalmer85/
> AnalysisTools","subtitle":"GitHub repository","main_image_url":"
> https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-
> 11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://
> cloud.githubusercontent.com/assets/143418/15842166/
> 7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in
> GitHub","url":"https://github.com/capalmer85/AnalysisTools"}
> },"updates":{"snippets":[{"icon":"PERSON","message":"@capalmer85 in #203:
> @mastrolorenzo I changed a few things about your implementation of the
> electroweak corrections. The biggest thing that you should note is that you
> should never add outfile-\u003eFill() or add a SetBranch(). If you want to
> create a new branch, you just need to add a line in the cfg/newbranches.txt
> file. Please look over this and let me know if you have some questions
> about this implementation.\r\n\r\nBtw, I've tested. It works, and I'm going
> to merge."}],"action":{"name":"View Pull Request","url":"https://
> github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373884628"}}}
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373906912>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFKqh1ey9KLgiTa4C_HLYo3zY83vi82Rks5tfNe_gaJpZM4SundU>
> .
>

capalmer commented:
I'm not sure how I got on this thread, but I've never worked on this
project. Looks like someone got the wrong email address. Good luck, though!


On Mar 17, 2018 12:55, "capalmer85" <notifications@github.com> wrote:

> Hi Luca. Very good. Btw, I think that your code shouldn't have worked at
> all because you re-declared the pointers locally in InitAnalysis(). Did
> you check to see the output of the function before you made the PR?
>
> On Sat, Mar 17, 2018 at 5:30 AM, mastrolorenzo <notifications@github.com>
> wrote:
>
> > hi @capalmer,Many thanks for the clarification about the branch
> > management! Concerning the modification you made, I have just went
> through
> > them in the code and it is perfectly fine for me.
> > cheersLuca
> >
> >
> >
> > Sent from my Samsung Galaxy smartphone.
> > -------- Original message --------From: capalmer85 <
> > notifications@github.com> Date: 17/03/2018 02:37 (GMT+01:00) To:
> > capalmer85/AnalysisTools <AnalysisTools@noreply.github.com> Cc:
> > mastrolorenzo <luca.mastrolorenzo@cern.ch>, Mention <
> > mention@noreply.github.com> Subject: Re: [capalmer85/AnalysisTools]
> > re-write some of the ewk corrections and fix few other typos (#203)
> > @mastrolorenzo I changed a few things about your implementation of the
> > electroweak corrections. The biggest thing that you should note is that
> you
> > should never add outfile->Fill() or add a SetBranch(). If you want to
> > create a new branch, you just need to add a line in the
> cfg/newbranches.txt
> > file. Please look over this and let me know if you have some questions
> > about this implementation.
> > Btw, I've tested. It works, and I'm going to merge.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub, or mute the thread.
> >
> >
> >
> >
> >
> >
> >
> >
> > {"api_version":"1.0","publisher":{"api_key":"
> > 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> > {"external_key":"github/capalmer85/AnalysisTools","title":"capalmer85/
> > AnalysisTools","subtitle":"GitHub repository","main_image_url":"
> > https://cloud.githubusercontent.com/assets/
> 143418/17495839/a5054eac-5d88-
> > 11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://
> > cloud.githubusercontent.com/assets/143418/15842166/
> > 7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in
> > GitHub","url":"https://github.com/capalmer85/AnalysisTools"}
> > },"updates":{"snippets":[{"icon":"PERSON","message":"@capalmer85 in
> #203:
> > @mastrolorenzo I changed a few things about your implementation of the
> > electroweak corrections. The biggest thing that you should note is that
> you
> > should never add outfile-\u003eFill() or add a SetBranch(). If you want
> to
> > create a new branch, you just need to add a line in the
> cfg/newbranches.txt
> > file. Please look over this and let me know if you have some questions
> > about this implementation.\r\n\r\nBtw, I've tested. It works, and I'm
> going
> > to merge."}],"action":{"name":"View Pull Request","url":"https://
> > github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373884628"}}}
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/capalmer85/AnalysisTools/pull/
> 203#issuecomment-373906912>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/AFKqh1ey9KLgiTa4C_
> HLYo3zY83vi82Rks5tfNe_gaJpZM4SundU>
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373939766>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABsmaW8iJClrQCCU6U3VhgqHfJ8vAVSJks5tfU4FgaJpZM4SundU>
> .
>

mastrolorenzo commented:
Hi Chris,

     indeed I did check that the files and the histograms were opened 
and read before send the PR, however I couldn't check directly the 
function output because there was not the "GenVbosons_pt" branch, that 
was included i think with the #186...

cheers,

Luca


Il 17/03/2018 18:55, capalmer85 ha scritto:
> Hi Luca. Very good. Btw, I think that your code shouldn't have worked at
> all because you re-declared the pointers locally in InitAnalysis(). Did
> you check to see the output of the function before you made the PR?
>
> On Sat, Mar 17, 2018 at 5:30 AM, mastrolorenzo <notifications@github.com>
> wrote:
>
> > hi @capalmer,Many thanks for the clarification about the branch
> > management! Concerning the modification you made, I have just went 
> through
> > them in the code and it is perfectly fine for me.
> > cheersLuca
> >
> >
> >
> > Sent from my Samsung Galaxy smartphone.
> > -------- Original message --------From: capalmer85 <
> > notifications@github.com> Date: 17/03/2018 02:37 (GMT+01:00) To:
> > capalmer85/AnalysisTools <AnalysisTools@noreply.github.com> Cc:
> > mastrolorenzo <luca.mastrolorenzo@cern.ch>, Mention <
> > mention@noreply.github.com> Subject: Re: [capalmer85/AnalysisTools]
> > re-write some of the ewk corrections and fix few other typos (#203)
> > @mastrolorenzo I changed a few things about your implementation of the
> > electroweak corrections. The biggest thing that you should note is 
> that you
> > should never add outfile->Fill() or add a SetBranch(). If you want to
> > create a new branch, you just need to add a line in the 
> cfg/newbranches.txt
> > file. Please look over this and let me know if you have some questions
> > about this implementation.
> > Btw, I've tested. It works, and I'm going to merge.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub, or mute the thread.
> >
> >
> >
> >
> >
> >
> >
> >
> > {"api_version":"1.0","publisher":{"api_key":"
> > 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> > {"external_key":"github/capalmer85/AnalysisTools","title":"capalmer85/
> > AnalysisTools","subtitle":"GitHub repository","main_image_url":"
> > 
> https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-
> > 11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://
> > cloud.githubusercontent.com/assets/143418/15842166/
> > 7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in
> > GitHub","url":"https://github.com/capalmer85/AnalysisTools"}
> > },"updates":{"snippets":[{"icon":"PERSON","message":"@capalmer85 in 
> #203:
> > @mastrolorenzo I changed a few things about your implementation of the
> > electroweak corrections. The biggest thing that you should note is 
> that you
> > should never add outfile-\u003eFill() or add a SetBranch(). If you 
> want to
> > create a new branch, you just need to add a line in the 
> cfg/newbranches.txt
> > file. Please look over this and let me know if you have some questions
> > about this implementation.\r\n\r\nBtw, I've tested. It works, and 
> I'm going
> > to merge."}],"action":{"name":"View Pull Request","url":"https://
> > github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373884628"}}}
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > 
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373906912>,
> > or mute the thread
> > 
> <https://github.com/notifications/unsubscribe-auth/AFKqh1ey9KLgiTa4C_HLYo3zY83vi82Rks5tfNe_gaJpZM4SundU>
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373939766>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS0Z-9o3E0IkOwglBkAiW_VckHf22iQks5tfU4FgaJpZM4SundU>.
>

-- 
---------------------------------------------------------------
Dr. Luca Mastrolorenzo
CERN - CMS experiment

Physics Institute III A - RWTH Aachen
Otto-Blumenthal-Str., 52056 Aachen, Germany
Office: 26A 205 Tel.: +49 241 80 27316	

Centre Européen pour la Recherche Nucléaire
Route de Meyrin 385, 1217 Meyrin, Suisse
Office: 40/1-B32 Tel.: +41 22 76 71645
---------------------------------------------------------------

mastrolorenzo commented:
Sorry, I thought it was fault of my wird Samsung keybord so that 
capalmer85 became capalmer! :)


Il 17/03/2018 19:08, Chris Palmer ha scritto:
> I'm not sure how I got on this thread, but I've never worked on this
> project. Looks like someone got the wrong email address. Good luck, 
> though!
>
>
> On Mar 17, 2018 12:55, "capalmer85" <notifications@github.com> wrote:
>
> > Hi Luca. Very good. Btw, I think that your code shouldn't have worked at
> > all because you re-declared the pointers locally in InitAnalysis(). Did
> > you check to see the output of the function before you made the PR?
> >
> > On Sat, Mar 17, 2018 at 5:30 AM, mastrolorenzo 
> <notifications@github.com>
> > wrote:
> >
> > > hi @capalmer,Many thanks for the clarification about the branch
> > > management! Concerning the modification you made, I have just went
> > through
> > > them in the code and it is perfectly fine for me.
> > > cheersLuca
> > >
> > >
> > >
> > > Sent from my Samsung Galaxy smartphone.
> > > -------- Original message --------From: capalmer85 <
> > > notifications@github.com> Date: 17/03/2018 02:37 (GMT+01:00) To:
> > > capalmer85/AnalysisTools <AnalysisTools@noreply.github.com> Cc:
> > > mastrolorenzo <luca.mastrolorenzo@cern.ch>, Mention <
> > > mention@noreply.github.com> Subject: Re: [capalmer85/AnalysisTools]
> > > re-write some of the ewk corrections and fix few other typos (#203)
> > > @mastrolorenzo I changed a few things about your implementation of the
> > > electroweak corrections. The biggest thing that you should note is 
> that
> > you
> > > should never add outfile->Fill() or add a SetBranch(). If you want to
> > > create a new branch, you just need to add a line in the
> > cfg/newbranches.txt
> > > file. Please look over this and let me know if you have some questions
> > > about this implementation.
> > > Btw, I've tested. It works, and I'm going to merge.
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub, or mute the thread.
> > >
> > >
> > >
> > >
> > >
> > >
> > >
> > >
> > > {"api_version":"1.0","publisher":{"api_key":"
> > > 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> > > {"external_key":"github/capalmer85/AnalysisTools","title":"capalmer85/
> > > AnalysisTools","subtitle":"GitHub repository","main_image_url":"
> > > https://cloud.githubusercontent.com/assets/
> > 143418/17495839/a5054eac-5d88-
> > > 11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://
> > > cloud.githubusercontent.com/assets/143418/15842166/
> > > 7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in
> > > GitHub","url":"https://github.com/capalmer85/AnalysisTools"}
> > > },"updates":{"snippets":[{"icon":"PERSON","message":"@capalmer85 in
> > #203:
> > > @mastrolorenzo I changed a few things about your implementation of the
> > > electroweak corrections. The biggest thing that you should note is 
> that
> > you
> > > should never add outfile-\u003eFill() or add a SetBranch(). If you 
> want
> > to
> > > create a new branch, you just need to add a line in the
> > cfg/newbranches.txt
> > > file. Please look over this and let me know if you have some questions
> > > about this implementation.\r\n\r\nBtw, I've tested. It works, and I'm
> > going
> > > to merge."}],"action":{"name":"View Pull Request","url":"https://
> > > 
> github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373884628"}}}
> > >
> > > —
> > > You are receiving this because you modified the open/close state.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/capalmer85/AnalysisTools/pull/
> > 203#issuecomment-373906912>,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/AFKqh1ey9KLgiTa4C_
> > HLYo3zY83vi82Rks5tfNe_gaJpZM4SundU>
> > > .
> > >
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > 
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373939766>,
> > or mute the thread
> > 
> <https://github.com/notifications/unsubscribe-auth/ABsmaW8iJClrQCCU6U3VhgqHfJ8vAVSJks5tfU4FgaJpZM4SundU>
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373940766>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS0Z45lfiDt1JlvjQCmBqHU96S_AQBpks5tfVETgaJpZM4SundU>.
>

-- 
---------------------------------------------------------------
Dr. Luca Mastrolorenzo
CERN - CMS experiment

Physics Institute III A - RWTH Aachen
Otto-Blumenthal-Str., 52056 Aachen, Germany
Office: 26A 205 Tel.: +49 241 80 27316	

Centre Européen pour la Recherche Nucléaire
Route de Meyrin 385, 1217 Meyrin, Suisse
Office: 40/1-B32 Tel.: +41 22 76 71645
---------------------------------------------------------------

mastrolorenzo commented:
@capalmer sorry, I thought it was fault of my wird Samsung keybord so 
that capalmer85 became capalmer! :)


Il 17/03/2018 19:08, Chris Palmer ha scritto:
> I'm not sure how I got on this thread, but I've never worked on this
> project. Looks like someone got the wrong email address. Good luck, 
> though!
>
>
> On Mar 17, 2018 12:55, "capalmer85" <notifications@github.com> wrote:
>
> > Hi Luca. Very good. Btw, I think that your code shouldn't have worked at
> > all because you re-declared the pointers locally in InitAnalysis(). Did
> > you check to see the output of the function before you made the PR?
> >
> > On Sat, Mar 17, 2018 at 5:30 AM, mastrolorenzo 
> <notifications@github.com>
> > wrote:
> >
> > > hi @capalmer,Many thanks for the clarification about the branch
> > > management! Concerning the modification you made, I have just went
> > through
> > > them in the code and it is perfectly fine for me.
> > > cheersLuca
> > >
> > >
> > >
> > > Sent from my Samsung Galaxy smartphone.
> > > -------- Original message --------From: capalmer85 <
> > > notifications@github.com> Date: 17/03/2018 02:37 (GMT+01:00) To:
> > > capalmer85/AnalysisTools <AnalysisTools@noreply.github.com> Cc:
> > > mastrolorenzo <luca.mastrolorenzo@cern.ch>, Mention <
> > > mention@noreply.github.com> Subject: Re: [capalmer85/AnalysisTools]
> > > re-write some of the ewk corrections and fix few other typos (#203)
> > > @mastrolorenzo I changed a few things about your implementation of the
> > > electroweak corrections. The biggest thing that you should note is 
> that
> > you
> > > should never add outfile->Fill() or add a SetBranch(). If you want to
> > > create a new branch, you just need to add a line in the
> > cfg/newbranches.txt
> > > file. Please look over this and let me know if you have some questions
> > > about this implementation.
> > > Btw, I've tested. It works, and I'm going to merge.
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub, or mute the thread.
> > >
> > >
> > >
> > >
> > >
> > >
> > >
> > >
> > > {"api_version":"1.0","publisher":{"api_key":"
> > > 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> > > {"external_key":"github/capalmer85/AnalysisTools","title":"capalmer85/
> > > AnalysisTools","subtitle":"GitHub repository","main_image_url":"
> > > https://cloud.githubusercontent.com/assets/
> > 143418/17495839/a5054eac-5d88-
> > > 11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://
> > > cloud.githubusercontent.com/assets/143418/15842166/
> > > 7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in
> > > GitHub","url":"https://github.com/capalmer85/AnalysisTools"}
> > > },"updates":{"snippets":[{"icon":"PERSON","message":"@capalmer85 in
> > #203:
> > > @mastrolorenzo I changed a few things about your implementation of the
> > > electroweak corrections. The biggest thing that you should note is 
> that
> > you
> > > should never add outfile-\u003eFill() or add a SetBranch(). If you 
> want
> > to
> > > create a new branch, you just need to add a line in the
> > cfg/newbranches.txt
> > > file. Please look over this and let me know if you have some questions
> > > about this implementation.\r\n\r\nBtw, I've tested. It works, and I'm
> > going
> > > to merge."}],"action":{"name":"View Pull Request","url":"https://
> > > 
> github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373884628"}}}
> > >
> > > —
> > > You are receiving this because you modified the open/close state.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/capalmer85/AnalysisTools/pull/
> > 203#issuecomment-373906912>,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/AFKqh1ey9KLgiTa4C_
> > HLYo3zY83vi82Rks5tfNe_gaJpZM4SundU>
> > > .
> > >
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > 
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373939766>,
> > or mute the thread
> > 
> <https://github.com/notifications/unsubscribe-auth/ABsmaW8iJClrQCCU6U3VhgqHfJ8vAVSJks5tfU4FgaJpZM4SundU>
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373940766>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS0Z45lfiDt1JlvjQCmBqHU96S_AQBpks5tfVETgaJpZM4SundU>.
>

-- 
---------------------------------------------------------------
Dr. Luca Mastrolorenzo
CERN - CMS experiment

Physics Institute III A - RWTH Aachen
Otto-Blumenthal-Str., 52056 Aachen, Germany
Office: 26A 205 Tel.: +49 241 80 27316	

Centre Européen pour la Recherche Nucléaire
Route de Meyrin 385, 1217 Meyrin, Suisse
Office: 40/1-B32 Tel.: +41 22 76 71645
---------------------------------------------------------------

capalmer commented:
Hehe no problem at all.


On Mar 17, 2018 13:14, "mastrolorenzo" <notifications@github.com> wrote:

> @capalmer sorry, I thought it was fault of my wird Samsung keybord so
> that capalmer85 became capalmer! :)
>
>
> Il 17/03/2018 19:08, Chris Palmer ha scritto:
> > I'm not sure how I got on this thread, but I've never worked on this
> > project. Looks like someone got the wrong email address. Good luck,
> > though!
> >
> >
> > On Mar 17, 2018 12:55, "capalmer85" <notifications@github.com> wrote:
> >
> > > Hi Luca. Very good. Btw, I think that your code shouldn't have worked
> at
> > > all because you re-declared the pointers locally in InitAnalysis(). Did
> > > you check to see the output of the function before you made the PR?
> > >
> > > On Sat, Mar 17, 2018 at 5:30 AM, mastrolorenzo
> > <notifications@github.com>
> > > wrote:
> > >
> > > > hi @capalmer,Many thanks for the clarification about the branch
> > > > management! Concerning the modification you made, I have just went
> > > through
> > > > them in the code and it is perfectly fine for me.
> > > > cheersLuca
> > > >
> > > >
> > > >
> > > > Sent from my Samsung Galaxy smartphone.
> > > > -------- Original message --------From: capalmer85 <
> > > > notifications@github.com> Date: 17/03/2018 02:37 (GMT+01:00) To:
> > > > capalmer85/AnalysisTools <AnalysisTools@noreply.github.com> Cc:
> > > > mastrolorenzo <luca.mastrolorenzo@cern.ch>, Mention <
> > > > mention@noreply.github.com> Subject: Re: [capalmer85/AnalysisTools]
> > > > re-write some of the ewk corrections and fix few other typos (#203)
> > > > @mastrolorenzo I changed a few things about your implementation of
> the
> > > > electroweak corrections. The biggest thing that you should note is
> > that
> > > you
> > > > should never add outfile->Fill() or add a SetBranch(). If you want to
> > > > create a new branch, you just need to add a line in the
> > > cfg/newbranches.txt
> > > > file. Please look over this and let me know if you have some
> questions
> > > > about this implementation.
> > > > Btw, I've tested. It works, and I'm going to merge.
> > > >
> > > > —
> > > > You are receiving this because you were mentioned.
> > > > Reply to this email directly, view it on GitHub, or mute the thread.
> > > >
> > > >
> > > >
> > > >
> > > >
> > > >
> > > >
> > > >
> > > > {"api_version":"1.0","publisher":{"api_key":"
> > > > 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> > > > {"external_key":"github/capalmer85/AnalysisTools","
> title":"capalmer85/
> > > > AnalysisTools","subtitle":"GitHub repository","main_image_url":"
> > > > https://cloud.githubusercontent.com/assets/
> > > 143418/17495839/a5054eac-5d88-
> > > > 11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://
> > > > cloud.githubusercontent.com/assets/143418/15842166/
> > > > 7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in
> > > > GitHub","url":"https://github.com/capalmer85/AnalysisTools"}
> > > > },"updates":{"snippets":[{"icon":"PERSON","message":"@capalmer85 in
> > > #203:
> > > > @mastrolorenzo I changed a few things about your implementation of
> the
> > > > electroweak corrections. The biggest thing that you should note is
> > that
> > > you
> > > > should never add outfile-\u003eFill() or add a SetBranch(). If you
> > want
> > > to
> > > > create a new branch, you just need to add a line in the
> > > cfg/newbranches.txt
> > > > file. Please look over this and let me know if you have some
> questions
> > > > about this implementation.\r\n\r\nBtw, I've tested. It works, and I'm
> > > going
> > > > to merge."}],"action":{"name":"View Pull Request","url":"https://
> > > >
> > github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373884628"}}}
> > > >
> > > > —
> > > > You are receiving this because you modified the open/close state.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/capalmer85/AnalysisTools/pull/
> > > 203#issuecomment-373906912>,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-
> auth/AFKqh1ey9KLgiTa4C_
> > > HLYo3zY83vi82Rks5tfNe_gaJpZM4SundU>
> > > > .
> > > >
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > >
> > <https://github.com/capalmer85/AnalysisTools/pull/
> 203#issuecomment-373939766>,
> > > or mute the thread
> > >
> > <https://github.com/notifications/unsubscribe-auth/
> ABsmaW8iJClrQCCU6U3VhgqHfJ8vAVSJks5tfU4FgaJpZM4SundU>
> > > .
> > >
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/capalmer85/AnalysisTools/pull/
> 203#issuecomment-373940766>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AFS0Z45lfiDt1JlvjQCmBqHU96S_AQBpks5tfVETgaJpZM4SundU>.
> >
>
> --
> ---------------------------------------------------------------
> Dr. Luca Mastrolorenzo
> CERN - CMS experiment
>
> Physics Institute III A - RWTH Aachen
> Otto-Blumenthal-Str., 52056 Aachen, Germany
> Office: 26A 205 Tel.: +49 241 80 27316 <+49%20241%208027316>
>
> Centre Européen pour la Recherche Nucléaire
> Route de Meyrin 385
> <https://maps.google.com/?q=Route+de+Meyrin+385&entry=gmail&source=g>,
> 1217 Meyrin, Suisse
> Office: 40/1-B32 Tel.: +41 22 76 71645
> ---------------------------------------------------------------
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/capalmer85/AnalysisTools/pull/203#issuecomment-373941177>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABsmaSdKVnD8AgyPsZEWgZRbDh5ka9D8ks5tfVJrgaJpZM4SundU>
> .
>
